### PR TITLE
initial commit of proposal 0153 add backslash character as operator head

### PR DESCRIPTION
**Pull request by nicoFe**

### CODE REVIEW

File: proposals/0153-add_backslash_character_as_operator-head.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+Swift-evolution thread: [Discussion thread topic for that proposal](https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20170130/031461.html)
+
+## Motivation
+
+The "problem" was identified while developing some code during linear algebra as
+I attempted to define the '\' operator as in MATLAB/Scilab/Octave (i.e. to solve 
+linear systems of equations: A\B means "solve the linear system A*X=B for X"). 
+
+Since the '\' character is not part of the swift grammar (expect within string
```
Comment (amlcurran) [inline proposals/0153-add_backslash_character_as_operator-head.md]: small typo - _expect_ should be _except_

Comment (nicoFe) [inline proposals/0153-add_backslash_character_as_operator-head.md]: Thanks. I just corrected it.

**Comment by JustinJiaDev** [comment]
Don't forget to change feature name (first line).

**Comment by DougGregor** [comment]
The core team discussed this proposal and determined that it doesn't meet the goals for Swift 4: it is purely additive (and can be added at any time), has no ABI or standard library impact, and there is a minor concern that `\` might be needed as a meta-escaping character somewhere in the future of Swift.

**Pull request closed without merging**